src/utils/getValidImageName.ts

```diff
@@ -16,5 +16,9 @@ export function getValidImageNameFromPath(appPath: string, tag?: string): string
 }
 
 export function getValidImageNameWithTag(nameHint: string, tag: string): string {
-    return `${getValidImageName(nameHint)}:${tag}`;
+    return getImageNameWithTag(getValidImageName(nameHint), tag);
+}
+
+export function getImageNameWithTag(nameHint: string, tag: string): string {
+    return `${nameHint}:${tag}`;

```

Comment:
Since Dotnet SDK in handling the image naming, we don't have to normalize it anymore. Instead, we'll just convert it to the image name format "name:tag"